src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -94,7 +94,7 @@ private class ACCT_IndividualAccounts_TEST {
     	//Set Individual (Bucket) settings
     	npe01__Contacts_And_Orgs_Settings__c contactsSettings = UTIL_CustomSettingsFacade.getOrgContactsSettings();
     	contactsSettings.npe01__Account_Processor__c = CAO_Constants.BUCKET_PROCESSOR;
-        update contactsSettings;
+        
```

Comment:
is the return value from getOrgContactSettings an object that all code will use the same instance of during tests?